.gitignore

```diff
@@ -2,4 +2,5 @@
 /dist
 /*.sh
 /venv*
-
```

Comment:
I often leave a single trailing newline because GTK-based editors on Fedora 33+ sometimes do not show the last line - but no hard feelings about this. So far this was just me fixing a client's problem. I can adapt when others start contributing.

Reply:
that is actually strange - locally my .gitignore file has a trailing newline - I'm confused ...

```
C:\Users\barsch\git\mjml-stub>git status
On branch more_tags
Your branch is up to date with 'origin/more_tags'.

nothing to commit, working tree clean

C:\Users\barsch\git\mjml-stub>more .gitignore
/build
/dist
/*.sh
/venv*
/.project
*.pyc

C:\Users\barsch\git\mjml-stub>
```

Reply:
well, not that important I guess.